### DISCUSSION TITLE
auto import client only component

### DISCUSSION THREAD

**Discussion by DeepKumbhare85** [nuxt/nuxt#22923] [Questions]
Hello ,

I have a component VueApexCharts.client.vue. code is like below

```vue 

<script setup lang="ts">
import VueApexCharts from 'vue3-apexcharts'
</script>

<template>
  <VueApexCharts />
</template>
```
And I use this component in my project whenever I require Apex chart.This works completely fine.
But is there any better way so that I can only use VueApexChart to render only on client side and without defining this component??

**Comment by olivervorasai** [comment]
Using `<ClientOnly>` as a wrapper around anything you want to only render on the client-side.
https://nuxt.com/docs/api/components/client-only

```.vue
pages/some-page.vue

<script setup lang="ts">
import VueApexCharts from 'vue3-apexcharts'
</script>

<template>
  <div>
    <h1>Hello this is a page</h1>
    <p>Look at my chart:</p>

    <ClientOnly>
      <VueApexCharts />
    </ClientOnly>
  <div/>
</template>
```

**Comment by DeepKumbhare85** [comment]
Hello @olivervorasai ,

Thank you for your reply. 

I want to import `VueApexCharts` for rendering on client-side only.Is it mandatory everytime to wrap VueApexCharts with `CientOnly` or is there any workaround in the nuxt such that whenever use `VueApexChart` , it will auto import to render on client side only??

 I just don't want to wrap `VueApexChart` everytime with `ClientOnly`.
